app/src/main/kotlin/no/nav/k9punsj/integrasjoner/gosys/GosysService.kt

```diff
@@ -0,0 +1,92 @@
+package no.nav.k9punsj.integrasjoner.gosys
+
+import net.logstash.logback.argument.StructuredArguments
+import no.nav.k9punsj.akjonspunkter.AksjonspunktService
+import no.nav.k9punsj.integrasjoner.pdl.PdlService
+import no.nav.k9punsj.journalpost.JournalpostService
+import no.nav.k9punsj.openapi.OasFeil
+import no.nav.k9punsj.tilgangskontroll.azuregraph.AzureGraphService
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import org.springframework.http.HttpStatus
+import org.springframework.http.MediaType
+import org.springframework.stereotype.Service
+import org.springframework.web.reactive.function.server.ServerResponse
+import org.springframework.web.reactive.function.server.bodyValueAndAwait
+import org.springframework.web.reactive.function.server.buildAndAwait
+import org.springframework.web.reactive.function.server.json
+
+@Service
+internal class GosysService(
+    private val pdlService: PdlService,
+    private val journalpostService: JournalpostService,
+    private val oppgaveGateway: OppgaveGateway,
+    private val aksjonspunktService: AksjonspunktService,
+    private val azureGraphService: AzureGraphService
+) {
+
+    private companion object {
+        val logger: Logger = LoggerFactory.getLogger(GosysService::class.java)
+    }
+
+    internal suspend fun opprettJournalforingsOppgave(
+        oppgaveRequest: GosysRoutes.GosysOpprettJournalføringsOppgaveRequest
+    ): ServerResponse {
```

Comment:
Usikker på om det blir riktig å returnere ServerResponse fr service klasse.
Tror det hadde vært bedre å kaste custom error/exception og håndtere det i routes, slik at den selv kan bestemme hvilket statuskoder og response den vil vise.

Reply:
Helt enig, men tenker att vi merger denne her och begynner testing så tar jag och begynner jobbet med journalpostroutes der vi returnerer riktig. Så får vi till en mall for hur resten skall se ut?

Reply:
Ok, det høres bra ut 👍